packages/thirdweb/package.json

```diff
@@ -217,6 +181,7 @@
     "fuse.js": "7.0.0",
     "input-otp": "^1.4.1",
     "mipd": "0.0.7",
+    "ox": "^0.1.1",
     "uqr": "0.1.2",
     "viem": "2.21.42"
```

Comment:
update `viem` to correspond to the same `ox` version otherwise we'll end up with duplicate dependencies